testing/infrastructure/InfrastructureComponentShape.ttl

```diff
@@ -31,7 +31,7 @@ shapes:InfrastructureComponentShape
 	    [ sh:class ids:Participant ; sh:order 2 ; ]
 	    [	sh:nodeKind sh:IRI ; sh:order 1 ; ]
 		) ;
-    sh:minCount 1 ;
+    #sh:minCount 1 ;
```

Comment:
I still recommend to not keep commented-out code as it confuses future development. IMHO we should remove it instead, because we can still access old stuff from the Git log, if needed.